src/braket/jobs/local/local_job_container.py

```diff
@@ -60,21 +60,22 @@ def __enter__(self):
         self._container_name = self._start_container(self.image_uri, self._force_update)
         return self
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
+    def __exit__(self, exc_type, exc_val, exc_tb):  # noqa: ANN001
```

Comment:
Same here, should we be adding types for this instead?

Reply:
This is added as a baseline - these types will be documented later.